guide/resources/chunk.xsl

```diff
@@ -1,5 +1,5 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version='1.0'>
-    <xsl:import href="http://docbook.sourceforge.net/release/xsl/current/xhtml/chunk.xsl"/>
+    <xsl:import href="http://docbook.sourceforge.net/release/xsl-ns/current/xhtml/chunk.xsl"/>
```

Comment:
You should switch to the new canonical URL: http://cdn.docbook.org/release/xsl/current/

Reply:
I used the old URL for backwards compatibility as the new one is only in use as of docbook-xsl 1.79.2. As we use Linux on Travis to validate the guide with an older version of docbook-xsl, there would be no catalog entry for this new URL.